thyrel-web/src/components/canvas/useCanvasPaint.tsx

```diff
@@ -153,4 +153,19 @@ function useCanvasPaint({
   };
 }
 
+function asyncWithPromise() {
+  return new Promise(r => {
+    setTimeout(() => {
+      r('vlaue');
```

Comment:
i guess it's 'value' ^^

Reply:
fallait me dire de supprimer cette exemple 😢 TU as très raison ce code ne doit pas être là